cloud/blockstore/vhost-server/server_ut.cpp

```diff
@@ -168,6 +168,21 @@ struct TFixture
         Files.clear();
         Options.Layout.clear();
     }
+
+    TSimpleStats GetStats(ui64 expectedCompleted) const
```

Comment:
let's add comment to this function that explains why we retry get stats.
Something similar to
"Without running io's stats are synced every second and only if there is a pending GetStats call. First call to GetStats might not bring the latest stats so we need at  least 2 calls so that the aio backend will sync the stats"